app/controllers/catalog_controller.rb

```diff
@@ -678,6 +678,39 @@ def process_sts(sts, task, _display_name = nil)
   end
   hide_action :process_sts
 
+  def template_to_node_name(object)
```

Comment:
@lgalis Could this method and the one below use a constant hash and just look up the values in it one way and use the inverted hash the other way?  Seems the methods would then just be one liners.